src/Me/components/Card/index.js

```diff
@@ -36,13 +19,41 @@ const EditButton = styled.button`
   cursor: pointer;
 `;
 
-const Content = styled.div`
+export const Content = styled.div`
```

Comment:
Is it used outside this file?

```suggestion
const Content = styled.div`
```

Reply:
.yes. In `MentroshipReq.js`

Reply:
Feels like `Content` is implementation details therefor not been exposed outside.
Can we make it configurable so the `Card` consumer can set if `Content` should has padding or not?
Can you see a quick fix here?

Reply:
I have to disagree with you on this one.  ❤
Essentially all I'm really doing here is changing some element style base on class hierarchy, completely normal.
`.root {
   .cls.cart-content {
     padding: 0
   }
}`

Adding new props to Card seems like overkill.

Reply:
I'm not sure it's completely normal. Think of `Card` as an external component. There is no option to override its style so you'd have to ask the maintainer to make it configurable.
This is in theory of course.

Reply:
Bottom line, we can merge it this way and fix it later if needed.